auacm/angular/src/app/models/profile-picture.ts

```diff
@@ -0,0 +1,421 @@
+import { ElementRef } from '@angular/core';
+
+const SELECTION_RADIUS_MIN = 64;
+const PROFILE_PIC_DIMENSION = 256;
+
+const MAX_SELECTED_IMG_WIDTH: number = document.documentElement.clientWidth * 0.65;
```

Comment:
As discussed, we decided that this isn't really a priority... Unless someone else thinks it should be.